### PR TITLE
Add array presenter

### PR DESCRIPTION
**Pull request by artyom-savchenko**
Signed-off-by: Artyom Savchenko <artem.savchenko@xored.com>

### CODE REVIEW

**Comment by z2sx** [comment]
This looks good as initial exemplary implementation. Probably, we'd be interested in parametrized `Presenters` e.g. `ArrayOf<Comment>` but this does not fit current infrastructure well and a subject to think well about.

File: plugins/presentation/src/components/internal/presenters/value/ArrayPresenter.svelte [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<style lang="scss">
+  .container {
+    display: flex;
+    flex-direction: column;
+  }
+</style>
+
+<div class="container">
+  {#each value as item (item.key)}
+    {value}
```
Comment (haiodo) [inline plugins/presentation/src/components/internal/presenters/value/ArrayPresenter.svelte:31]: Probable as first implementation we could go with choose a presenter based on value._class?

Comment (z2sx) [reply plugins/presentation/src/components/internal/presenters/value/ArrayPresenter.svelte:31]: > Probable as first implementation we could go with choose a presenter based on value._class?

Sounds reasonable, remaining question is how we arrange them as an array :)

**Comment by artyom-savchenko** [comment]
@z2sx @haiodo  I updated the PR, please take a look.
Below is an example of displaying comments array:
Screen Shot 2021-03-13 at 13 21 13

**Pull request merged**